soh/src/overlays/actors/ovl_En_Go2/z_en_go2.c

```diff
@@ -346,7 +346,7 @@ s16 EnGo2_GetStateGoronCityRollingBig(GlobalContext* globalCtx, EnGo2* this) {
                     if(!gSaveContext.n64ddFlag) {
                         bombBagUpgrade = CUR_CAPACITY(UPG_BOMB_BAG) == 30 ? GI_BOMB_BAG_40 : GI_BOMB_BAG_30;    
                     } else {
-                        bombBagUpgrade = Randomizer_GetItemIdFromKnownCheck(RC_GC_ROLLING_GORON_AS_CHILD, GI_BOMB_BAG_40);
+                        bombBagUpgrade = Randomizer_GetItemFromKnownCheck(RC_GC_ROLLING_GORON_AS_CHILD, GI_BOMB_BAG_40).getItemId;
                     }
                     EnGo2_GetItem(this, globalCtx, bombBagUpgrade);
```

Comment:
`EnGo2_GetItem` needs to be looked at, and possibly changed to take the getItemEntry rather than just the getItemId

Reply:
Looks like EnGo2_GetItem is just the give func but with some specifics on the range, gonna copy it